packages/context-editor/src/schemas/strike.ts

```diff
@@ -0,0 +1,29 @@
+import type { DOMOutputSpec, MarkSpec } from "prosemirror-model";
+
+const getAttrs = (node: HTMLElement) => {
+	return {
+		id: (node as Element).getAttribute("id"),
+		class: (node as Element).getAttribute("class"),
+	};
+};
+
+export default {
+	attrs: {
+		id: { default: null },
+		class: { default: null },
+	},
+	parseDOM: [
+		{ tag: "s", getAttrs },
+		{ tag: "strike", getAttrs },
+		{ tag: "del", getAttrs },
```

Comment:
according to mdn `<del>` is used to mark a deletion in a diff (i.e. track changes/suggested edits), which we might eventually support. so we may not want to coerce those to `<s>` elements, which would happen if you pasted content with a `<del>` element in. but this is probably fine for now!

Reply:
oh that's good to know! I copied these from legacy but what you said makes sense. I'll keep it in mind for the future!